features/0794-did-rotate/README.md

```diff
@@ -0,0 +1,147 @@
+# Aries RFC 0794: DID Rotate 1.0
+
+- Authors: [Sam Curren](telegramsam@gmail.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2023-08-29 
+- Status Note: 
+- Start Date: 2023-08-18 
+- Tags: [feature](/tags.md#feature), [protocol](/tags.md#protocol)
+- URI: https://didcomm.org/did-rotate/1.0
+
+## Summary
+
+This protocol signals the change of DID in use between parties.
+
+This protocol is only applicable to DIDComm v1 - in DIDComm v2 use the more efficient DID Rotation header.
+
+## Motivation
+
+This mechanism allows a party in a relationship to change the DID they use to identify themselves in that relationship. This may be used to switch DID methods,
+but also to switch to a new DID within the same DID method. For non-updatable DID methods, this allows updating DID Doc attributes such as service endpoints. Inspired by (but different from) the DID rotation feature of the DIDComm Messaging (DIDComm v2) spec.
```

Comment:
I think updating DID Doc attributes for non-resolvable DIDs would need to also attach the DID Document, like we do in DID Exchange protocol.

Do you think we can add an attachment for those cases to allow a broader range of use cases of this protocol, or limit it to those DIDs that are publicly resolvable or embed the DID Document on its identifier?

Reply:
Given the timing of this and the availability/desirability of `did:peer:4`, that we should not add this. We don’t want implementations using either unqualified DIDs or `did:peer:1`, which are the only options AFAIK that would need this.  We could add a note about this, but I don’t think we should add the attachment option.

Reply:
I'm not sure I understand the question. Let's discuss in the Aries WG.

Reply:
per discussion in WG 20230920: Would prefer not to add support in favor of did:peer:4, but willing to add if this presents a significant hurdle for the community using did:peer:1.